cg/apps/demultiplex/sample_sheet/validate.py

```diff
@@ -62,7 +62,7 @@ def get_raw_samples(sample_sheet_content: List[List[str]]) -> List[Dict[str, str
         # Skip lines that are too short to contain samples
         if len(line) <= 5:
             continue
-        if line[0] == SampleSheetHeaderColumnNames.FLOW_CELL_ID:
+        if line[0] == SampleSheetNovaSeq6000Sections.Data.FLOW_CELL_ID.value:
```

Comment:
I don't think it is necessary to use `.value` for members of a string enum! So `SampleSheetNovaSeq6000Sections.Data.FLOW_CELL_ID` 😄 Same for all other instances of `.value` if it is a string enum.

Reply:
I have tried, and it fails without adding `.value`

Reply:
Hm, I must be misinformed then 😞 Sorry